osbs/api.py

```diff
@@ -381,10 +394,13 @@ def create_prod_build(self, git_uri, git_ref,
             git_push_url=self.build_conf.get_git_push_url(),
             git_push_username=self.build_conf.get_git_push_username(),
             builder_build_json_dir=self.build_conf.get_builder_build_json_store(),
-            scratch=scratch,
+            scratch=self.build_conf.get_scratch(),
```

Comment:
Careful here. This changes means that create_prod_build(..., scratch=True) will no longer have any effect. Instead, the scratch parameter would have to be passed to the Configuration() constructor, or put in a config file.

We could actually solve this problem by having get_scratch() take a parameter to use as the default, and giving it the 'scratch' we were passed here as its parameter.